template_smiles.h

```diff
@@ -91,7 +91,8 @@ const std::vector<std::string> TEMPLATE_SMILES = {
     "C1CC2NCC1CN2 |(0.18,3.02,;1.22,3.92,;0.63,5.1,;-0.75,4.3,;-0.75,2.69,;0.63,1.89,;2.02,2.69,;2.02,4.3,)|",
     "C1C2CC3CC1CC3C2 |(-7.01,3.13,;-7.71,4.35,;-7.01,5.56,;-5.61,5.56,;-4.91,4.35,;-5.61,3.13,;-4.28,3.57,;-4.28,5.13,;-6.34,4.05,)|",
     "C1CC2C3CCC4C1C1CCC2C2CCC1C1CCC2C3CCC41 |(-1.94049,-5.53747,;-1.94049,-1.8102,;-0.825758,-0.511159,;-1.57576,0.787879,;-3.07576,0.787879,;-3.07576,-2.93939,;-1.57576,-2.93939,;-0.825758,-4.23843,;0.674242,-4.23843,;1.72266,-5.53747,;1.72266,-1.8102,;0.674242,-0.511159,;1.42424,0.787879,;2.92424,0.787879,;2.92424,-2.93939,;1.42424,-2.93939,;0.674242,-1.64036,;1.72266,-0.341318,;1.72266,3.38596,;0.674242,2.08692,;-0.825758,2.08692,;-1.94049,3.38596,;-1.94049,-0.341318,;-0.825758,-1.64036,)|",
-    "C1C2C3CC4C5CC3C1C5C24 |(-1.68833,-1.6056,;-0.474804,-0.723925,;-1.13662,-0.251107,;0.360708,1.45266,;-0.93833,2.20266,;-2.43833,2.20266,;-3.73737,1.45266,;-2.19718,-0.251107,;-2.90185,-0.723925,;-2.43833,0.70266,;-0.93833,0.70266,)|",
     "C1CC2CC1C2 |(0.562,0.77,;0.562,-0.77,;-0.9027,-1.2459,;-1.8079,0,;-0.9027,1.2459,;-0.5912,0,)|",
+    "C1C2C3CC4C5CC3C1C5C24 |(-1.68833,-1.6056,;-0.474804,-0.723925,;-1.13662,-0.251107,;0.360708,1.45266,;-0.93833,2.20266,;-2.43833,2.20266,;-3.73737,1.45266,;-2.19718,-0.251107,;-2.90185,-0.723925,;-2.43833,0.70266,;-0.93833,0.70266,)|",
+
```

Comment:
If you did this to trigger the GH action -- I think you need to make the change in the .smi file and not the .h file

Reply:
oh I see, thanks! it should be fixed now